print/cups-ppdc/Portfile

```diff
@@ -0,0 +1,100 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem      1.0
+PortGroup       legacysupport 1.1
```

Comment:
why do you need to use the legacysupport PG? Typically there would be a comment for what missing function is needed and a specification on which systems